src/mugs.js

```diff
@@ -295,7 +295,44 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (mug.__className === 'Repeat') {
+                            return false;
+                        }
+                        return recFunc(mug.form.tree.getNodeFromMug(mug).parent.value);
```

Comment:
If not, is there any reason why we can't declare an invariant that they will always be the same?